src/main/java/uk/gov/hmcts/reform/migration/CaseMigrationProcessor.java

```diff
@@ -96,12 +167,14 @@ private void validateCaseType(String caseType) {
         }
     }
 
+
     private void updateCase(String authorisation, String caseType, CaseDetails caseDetails) {
         if (dataMigrationService.accepts().test(caseDetails)) {
             Long id = caseDetails.getId();
-            log.info("Updating case {}", id);
+            log.info("Updating case {} at state {}", id);
             try {
-                log.debug("Case data: {}", caseDetails.getData());
+                log.debug("Case data before: {}", caseDetails.getData());
```

Comment:
The entire case will be logged here - we must strive to keep sensitive data out of logs.